src/core/server/models/tenant/tenant.ts

```diff
@@ -102,6 +103,12 @@ export interface TenantSettings
    */
   featureFlags?: Array<GQLFEATURE_FLAG | LEGACY_FEATURE_FLAGS>;
 
+  /**
+   * flattenReplies is whether the tenant wants replies to be hidden behind
+   * a "Show more of this conversation" link.
+   */
+  flattenReplies?: boolean;
+
```

Comment:
I think we should add to this file instead: `src/core/server/models/settings/settings.ts` where the `Settings` type is exported. There you find all the other settings as well.